src/patternfly/components/AppLauncher/deprecated/application-launcher.md

```diff
@@ -13,10 +13,10 @@ import './application-launcher.css'
 ```hbs
 {{#> app-launcher app-launcher--id="application-launcher-collapsed"}}
   {{#> app-launcher-menu}}
-    <li>{{#> app-launcher-menu-item app-launcher-menu-item--attribute='href="#"'}}Link{{/app-launcher-menu-item}}</li>
-    <li>{{#> app-launcher-menu-item app-launcher-menu-item--type="button"}}Action{{/app-launcher-menu-item}}</li>
+    <li role="menuitem">{{#> app-launcher-menu-item app-launcher-menu-item--attribute='href="#"'}}Link{{/app-launcher-menu-item}}</li>
```

Comment:
Similar to the above comment, we'd want to place this role on `app-launcher-menu-item` instead, then have this `li` element be `role="none"`